### ISSUE TITLE
Add support for the new username system

### ISSUE THREAD

**Issue by among9** [Aliucord/Aliucord#359]
### Discord Account

_ZackYT#6316

### Plugin Description

Add the thing people did for like voice messages but with display names please it would be great

### Go into more detail...

have it so you can set display names see them and have a setting where it also shows the old system

### Request Agreement

- [X] I have made sure my plugin is possible and abides by the rules!

### ​

- [X] I did indeed check to make sure my plugin request is original!

**Comment by pp0236** [comment]
also add add friend by pomelo :)
![Screenshot_20230613-091421](https://github.com/Aliucord/Aliucord/assets/54238880/9d004130-26bf-4282-887b-8cec4c7ecb21)

**Comment by SavageTheUnicorn** [comment]
Yes, adding by pomelo still isn't working as far as I can tell. Unsure how to work around this.

**Comment by Kassims** [comment]
bump

**Comment by zt64** [comment]
Display names are supported, just not adding friends. I'll close this issue as #471 is specifically about sending friend requests.
